src/cmd/flux-job.c

```diff
@@ -2713,6 +2713,19 @@ int cmd_wait_event (optparse_t *p, int argc, char **argv)
     return (0);
 }
 
+void info_list_keys (flux_t *h)
```

Comment:
Perhaps drop the unused flux_t argument and rename function to info_usage()?
O/w LGTM

Reply:
oh duh, yeah, i don't use that arg anymore :-)